pom.xml

```diff
@@ -217,7 +217,8 @@
     <testsThreadCount>4</testsThreadCount>
 
     <!--metrics-->
-    <dropwizard.version>3.2.5</dropwizard.version>
+    <dropwizard.version>4.2.9</dropwizard.version>
+    <dropwizard.ganglia.version>3.2.5</dropwizard.ganglia.version>
```

Comment:
@SpriCoder , Let's use 3.2.6?

Reply:
To specific, I want to make sure that you mean upgrade ganglia version to 3.2.6 or else?

Reply:
Yes, I mean upgrading ganglia version to 3.2.6

Reply:
Ok, I will update soon.

Reply:
I have already upgraded, Please approve the workflows, thanks for your time.